beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -647,6 +648,33 @@ impl<T: BeaconChainTypes> Worker<T> {
         }
     }
 
+    /// Process the beacon block received from the gossip network and:
+    ///
+    /// - If it passes gossip propagation criteria, tell the network thread to forward it.
+    /// - Attempt to add it to the beacon chain, informing the sync thread if more blocks need to
+    ///   be downloaded.
+    ///
+    /// Raises a log if there are errors.
```

Comment:
just a reminder that the comment needs update here, possibly when it's implemented

Reply:
I'll remove it